src/Microsoft.Health.Dicom.Core/Features/ChangeFeed/ChangeFeedAction.cs

```diff
@@ -9,4 +9,5 @@ public enum ChangeFeedAction
 {
     Create = 0,
     Delete = 1,
+    Update = 2,
```

Comment:
Maybe you can make this change later to when you are ready to fix the swagger check? Otherwise you will have to fix the check before checking this in

Reply:
Yeah, I am disabling the check now, will enable once we are done. That swagger breaking check is not that helpful